account_outstanding_payment/models/account_invoice.py

```diff
@@ -7,8 +7,9 @@
 from openerp.tools.translate import _
 import json
 from openerp.tools.float_utils import float_is_zero
+import logging
 
-
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
This change is out-of-scope of the description of the PR

Reply:
Yes, sorry I was testing and forgot to delete that line.